trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/CST/airfoil_from_CST.py

```diff
@@ -0,0 +1,137 @@
+## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil-CST
+# create_airfoil_from_CST.py
+# 
+# Created:  Sep 2019, S.Karpuk
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+from SUAVE.Core import Data
+
+import numpy as np
+from math import pi, cos, factorial
```

Comment:
don't use math functions. Instead rely on numpy. Math functions are generally slower and it's another import we need to track.